packages/mp/src/mp/validate/pre_build_validation.py

```diff
@@ -70,11 +79,104 @@ def run_pre_build_validation(self) -> None:
         )
 
     def _get_validation_functions(self) -> list[Callable]:
-        return [
-            self._uv_lock_validation,
-        ]
+        return [self._uv_lock_validation, self._version_bump_validation]
 
     def _uv_lock_validation(self) -> None:
         self.results.errors.append("[yellow]Running uv lock validation [/yellow]")
         if not mp.core.file_utils.is_built(self.integration_path):
             mp.core.unix.check_lock_file(self.integration_path)
+
+    def _version_bump_validation(self) -> None:
+        self.results.errors.append("[yellow]Running version bump validation [/yellow]")
+
+        head_sha = os.environ.get("GITHUB_SHA")
```

Comment:
```suggestion
        head_sha: str = os.environ.get("GITHUB_SHA")
```

Are we sure we'll always have this envar when running?